tests/conftest.py

```diff
@@ -140,40 +151,36 @@ def cluster_teardown_finalizer():
     with open(install_config, "w") as f:
         f.write(install_config_str)
 
-    # Download installer
-    installer = get_openshift_installer(
-        config.DEPLOYMENT['installer_version']
-    )
     # Download client
     get_openshift_client()
 
     # Deploy cluster
-    log.info("Deploying cluster")
-    run_cmd(
-        f"{installer} create cluster "
-        f"--dir {cluster_path} "
-        f"--log-level {log_cli_level}"
-    )
+    deployer.deploy(log_cli_level)
+    deployer.add_volume()
+
+
+@pytest.fixture(scope="session", autouse=True)
+def ocs_cluster(cluster):
+    """
+    This fixture is exclusively for OCS deployment.
+    No OCS teardown is done here
+    # TODO: figure out proper steps for only OCS teardown
+    we can skip ocs deployment using config.ENV_DATA['skip_ocs_deployment']
+    """
+
+    if config.ENV_DATA['skip_ocs_deployment']:
+        return
+    cluster_path = config.ENV_DATA['cluster_path']
+    _templating = templating.Templating()
+
+    try:
+        create_oc_resource('common.yaml', cluster_path, _templating, config.ENV_DATA)
```

Comment:
yep, this is not nice solution how to check if cluster exists.

Reply:
I think ```is_cluster_running() ``` is for OCP cluster, What we need here is for OCS cluster. I can't even rely on ```ceph_health_check``` because healtherror doesn't mean cluster doesn't exist. I am looking for verifying a ocp.get call on  'CephCluster' kind and check whether it returned something.

Reply:
Done, Now a check for CephCluster kind has been put in the code.